geoips/interfaces/yaml_based/gridline_annotators.py

```diff
@@ -11,5 +11,28 @@ class GridlineAnnotatorsInterface(BaseYamlInterface):
 
     name = "gridline_annotators"
 
+    # Keyword arguments that can be used to adjust the style of lat/lon labels applied
+    # to annotated imagery. See geoips.image_utils.maps:draw_gridlines for how this is
+    # implemented
+    label_kwargs = {
```

Comment:
When we switch to pydantic, can we allow yaml files to have any valid kwarg for the labels (rather than specify explicitly), and validate with a custom validator?

Reply:
OR could we expose something like https://matplotlib.org/stable/api/matplotlib_configuration_api.html#matplotlib.RcParams to yaml users?